### PR TITLE
Adding EDB BDR Redhat/rocky repo in the setup_repo based on the conditions

### PR DESCRIPTION
**Pull request by vibhorkum**

### CODE REVIEW

File: .ansible-lint [hunk 1]
```diff
@@ -1,3 +1,6 @@
+skip_list:
+- '503'
+- '403'
```
Comment (jt-edb) [inline .ansible-lint:3]: I think it's better to use `warn_list`.

File: roles/setup_repo/tasks/PG_RedHat_setuprepos.yml [hunk 2]
```diff
@@ -92,8 +92,43 @@
...
+      set -o pipefail;
+      curl -sS "{{ edb_2q_base_repo_link }}/harp/release/{{ pg_version }}/rpm" | bash
+  args:
+    executable: /bin/bash
+  register: reposub
+  failed_when: >
+    reposub.rc != 0 or 'error: ' in reposub.stdout.lower()
+  when: >
+    bdr_nodes_ips is defined and ansible_host in bdr_nodes_ips
+    and pg_version|int >= 13
```
Comment (jt-edb) [inline roles/setup_repo/tasks/PG_RedHat_setuprepos.yml:134]: What do think you about using a new variable containing the list of repositories and looping over this list:
```yaml
2nq_repositories:
- bdr/release
- harp/release
```

**Review by jt-edb — COMMENTED** [review]
Please check inline comments

File: .ansible-lint (hunk 1 above)
Comment (vibhorkumar123) [reply .ansible-lint:3]: Somehow warn_list was not working correctly for me, therefore added skip_list

File: roles/setup_repo/tasks/PG_RedHat_setuprepos.yml (hunk 2 above)
Comment (vibhorkumar123) [reply roles/setup_repo/tasks/PG_RedHat_setuprepos.yml]: Modified as per your suggestion. Please have a look.

**Review by jt-edb — COMMENTED** [review]
LGTM

**Pull request merged**